grpc/src/test/java/com/linecorp/armeria/internal/grpc/GrpcStatusTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright 2014, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.linecorp.armeria.internal.grpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import com.linecorp.armeria.common.HttpStatus;
+
+import io.grpc.Status;
+
+public class GrpcStatusTest {
+    @Test
+    public void fromGrpcStatusCode() {
```

Comment:
The conversion seems to be implemented in a library called `gax-java`. I think it's too heavyweight to import for the conversion itself, but we may as well use here it to make this test automatic / exhaustive.

```groovy
testCompile group: 'com.google.api', name: 'gax-grpc', version: '1.29.0'
```

```java
public class GrpcStatusTest {
    @Test
    public void fromGrpcStatusCode() {
        for (Status.Code code : Status.Code.values()) {
            assertThat(GrpcStatus.fromGrpcStatusCode(code).code())
                    .as("gRPC code: {}", code)
                    .isEqualTo(GrpcStatusCode.of(code).getCode().getHttpStatusCode());
        }
    }
}
```